package.json

```diff
@@ -64,7 +64,7 @@
     "yarn": "^1.12.0"
   },
   "scripts": {
-    "test": "gulp validate",
+    "test": "gulp validate && gulp axe",
```

Comment:
Why not add it to the validate task?

Reply:
Nevermind, guessing the precommit would take to long?
Or because you need to build first?

:+1:  carry on :smile: